docs/sections/cn_workflows.md

```diff
@@ -385,7 +385,22 @@ use the `--resource-manager` or `-r` option of the `popper run` subcommand to sp
 popper run -f sample.yml -r kubernetes
 ```
 
-For a workflow we build an image and create a volume in the namespace defined in the kubefig. This volume is the with a ReadWrite access.
+For a workflow we build an image and create a volume in the namespace defined in the [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/). 
```

Comment:
the narrative style of the documentation is in third person. So this would be "Given a workflow such as the one shown above, Popper builds an image and creates a volume..."

Reply:
Awesome! I'd take note of this